_includes/header.html

```diff
@@ -1,58 +1,54 @@
+  <meta name="msapplication-TileImage" content="{{site.baseurl}}/icons/ms-icon-144x144.png">
+  <meta name="theme-color" content="#ffffff">
+  <title>GSA - IT Modernization Centers of Excellence</title>
 
-{% assign description = page.description | default: site.description %}
-{% if description %}
-<meta name="description" content="{{ description }}">
-{% endif %}
-<meta property="og:image" content="{{site.baseurl}}/img/coe-logomark.png">
+  {% assign description = page.description | default: site.description %}
+  {% if description %}
+  <meta name="description" content="{{ description }}">
+  {% endif %}
+  <meta property="og:image" content="{{site.baseurl}}/img/coe-logomark.png">
 
-<link rel="stylesheet" href="{{site.baseurl}}/css/uswds.min.css">
-<link rel="stylesheet" href="{{site.baseurl}}/css/styles.css">
-<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-<!--[if lt IE 9]>
-  <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-  <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-<![endif]-->
+  <link rel="stylesheet" href="{{site.baseurl}}/css/uswds.min.css">
+  <link rel="stylesheet" href="{{site.baseurl}}/css/styles.css">
+  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+  <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+  <![endif]-->
 
-    <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-          new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-PBX86M3');</script>
-    <!-- End Google Tag Manager -->
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-PBX86M3');</script>
+  <!-- End Google Tag Manager -->
 
-    <script src="{{site.baseurl}}/js/jquery.min.js"></script>
-    <script src="{{site.baseurl}}/js/uswds.min.js"></script>
-    <script src="{{site.baseurl}}/js/scripts.js"></script>
+  <script src="{{site.baseurl}}/js/jquery.min.js"></script>
+  <script src="{{site.baseurl}}/js/uswds.min.js"></script>
+  <script src="{{site.baseurl}}/js/scripts.js"></script>
```

Comment:
Everything above this point is just showing up as changed because of the head/body tag cleanup I did. None of the actual links should have changed.